source/guides/stacks/mers.rst

```diff
@@ -176,7 +217,7 @@ can be used for reference.
 
```

Comment:
Line 152: General rule, don't use -ing words in section titles. Instead "Build the MeRS container image from source"